detection-rules/spam_free_tools_giveaway.yml

```diff
@@ -0,0 +1,91 @@
+name: "Spam: Free tools and giveaways by brands"
```

Comment:
call me crazy, but this detection logic has nothing to do with giveways, tools, or brands. 

I'm all for this rule (it performs well) but maybe change the name/description?